packages/fxa-settings/scripts/test-ci.sh

```diff
@@ -1,5 +1,31 @@
 #!/bin/bash -ex
 
-yarn workspaces focus fxa-settings
-PUBLIC_URL=/ INLINE_RUNTIME_CHUNK=false CI=false SKIP_PREFLIGHT_CHECK=true yarn build
 CI=yes SKIP_PREFLIGHT_CHECK=true yarn test
+
+mkdir -p config
+
+cd ../../
+mkdir -p ~/.pm2/logs
+mkdir -p artifacts/tests
+yarn workspaces foreach \
+    --verbose \
+    --topological-dev \
+    --include browserid-verifier \
+    --include fxa-auth-db-mysql \
+    --include fxa-auth-server \
+    --include fxa-content-server \
+    --include fxa-profile-server \
+    --include fxa-react \
+    --include fxa-settings \
+    --include fxa-shared \
+    --include fxa-graphql-api \
+    run start > ~/.pm2/logs/startup.log
+npx pm2 ls
```

Comment:
This just outputs the processes, is this just for logging purposes? Do we need this?

Reply:
I found it useful to verify that things are started as expected.

Reply:
No harm done! 👍